### PR TITLE
Add composition ADR

### PR DESCRIPTION
**Pull request by Hinton**
Prefer composition over inheritance

### CODE REVIEW

File: decisions/0009-angular-composition-over-inheritance.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Angular - Composition over inheritance
+
+## Context and Problem Statement
+
+We currently rely heavily on inheritance for our Angular applications. While this seemed like a
+natural decision at the time since it allowed us to quickly share code between different areas. It
+has also lead to tight coupling, difficulty in understanding the impact a change will have. It also
```
Comment (MGibson1) [inline decisions/0009-angular-composition-over-inheritance.md]:
```suggestion
has also lead to tight coupling and therefore difficulty in understanding the impact a change will have. It also
```

**Review by MGibson1 — CHANGES_REQUESTED** [review]
As is typical I think we need to treat this like a goal document until we're closer to this ideal.

Do you have an example component that is doing this well? Should we link some recent change as an example of stepping in this direction?

**Comment by Hinton** [comment]
I have examples on how to not do it 😁. No but I'll try and find some good examples.

**Comment by MGibson1** [comment]
Right, unfortunately right now we have almost exclusively anti-examples. I think we need a way to call that out in this (and probably other) ADRs.

Some kind of status like `In Progress` or `Standard in Code` that tells you if you can trust patterns you find in repositories we host

**Review by MGibson1 — APPROVED** [review]
I want to get this into ratification to encourage improvement, but I think there's still an opportunity for examples and help links on how to progress from our current state to a `standard` status.

**Pull request merged**